tests/v2/validation/rbac/rbac.go

```diff
@@ -237,6 +238,19 @@ func createRole(client *rancher.Client, context string, roleName string, rules [
 
 }
 
+func CheckRBACResourceExistsForUser(client *rancher.Client, collection *v1.SteveCollection, userName string) (bool, error) {
```

Comment:
Can we make this an extension? If the rbac tests are moved to integration suite, we won't be able to access the helper and adding an extension would help us not change anything..

Reply:
Moved this function to rbac_test.go instead based on your review comment below